ocs_ci/ocs/node.py

```diff
@@ -2513,9 +2513,11 @@ def check_for_zombie_process_on_node(node_name=None):
     for node_obj in node_obj_list:
         debug_cmd = (
             f"debug nodes/{node_obj.name} --to-namespace={config.ENV_DATA['cluster_namespace']} "
-            '-- chroot /host /bin/bash -c "ps -A -ostat,pid,ppid | grep -e "[zZ]""'
+            '-- chroot /host /bin/bash -c "ps -A -ostat,pid,ppid | grep -e [zZ]"'
+        )
+        out = node_obj.ocp.exec_oc_cmd(
+            command=debug_cmd, ignore_error=True, out_yaml_format=False
```

Comment:
Yes. if command dont found zombie process then it will send error. As We want to prioceed if zombie process not found. thats why ignoring error.
I have added ignore_error=True so it will not raise error but will send TRue or False. depends on true and false i added copde below in if else.

Our discussion- https://chat.google.com/dm/qtdUE4AAAAE/rnKiKmaDL1I/rnKiKmaDL1I?cls=10